tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/DataApiController.cs

```diff
@@ -19,6 +20,12 @@ public IActionResult Model(MyModel model)
             return Content($"Received model with properties: {model}");
         }
 
+        [Route("array")]
+        public IActionResult Array(IEnumerable<string> model)
```

Comment:
Is this controller action used anywhere?

Reply:
it was mostly used manually to test different object types